data-prepper-plugins/http-source/src/main/java/org/opensearch/dataprepper/plugins/source/loghttp/HTTPSourceConfig.java

```diff
@@ -154,75 +43,7 @@ public String getPath() {
         return path;
     }
 
-    public int getRequestTimeoutInMillis() {
-        return requestTimeoutInMillis;
-    }
-
-    public int getBufferTimeoutInMillis() {
-        return (int)(BUFFER_TIMEOUT_FRACTION * requestTimeoutInMillis);
-    }
-
-    public int getThreadCount() {
-        return threadCount;
-    }
-
-    public int getMaxConnectionCount() {
-        return maxConnectionCount;
-    }
-
-    public int getMaxPendingRequests() {
-        return maxPendingRequests;
-    }
-
-    public boolean isSsl() {
-        return ssl;
-    }
-
-    public String getSslCertificateFile() {
-        return sslCertificateFile;
-    }
-
-    public String getSslKeyFile() {
-        return sslKeyFile;
-    }
-
-    public boolean isUseAcmCertificateForSsl() {
-        return useAcmCertificateForSsl;
-    }
-
-    public String getAcmCertificateArn() {
-        return acmCertificateArn;
-    }
-
-    public String getAcmPrivateKeyPassword() {
-        return acmPrivateKeyPassword;
-    }
-
-    public int getAcmCertificateTimeoutMillis() {
-        return acmCertificateTimeoutMillis;
-    }
-
-    public String getAwsRegion() {
-        return awsRegion;
-    }
-
-    public PluginModel getAuthentication() {
-        return authentication;
-    }
-
-    public boolean hasHealthCheckService() {
-        return healthCheckService;
-    }
-
-    public boolean isUnauthenticatedHealthCheck() {
-        return unauthenticatedHealthCheck;
-    }
-
     public CompressionOption getCompression() {
```

Comment:
Do we need to add this to `HttpServerConfig`? How will it be used if it is only here?